docs/source/tutorials/running_a_simulation/interactive.rst

```diff
@@ -0,0 +1,631 @@
+
+.. code-block:: python
+   from vivarium import initialize_simulation_from_model_specification
+   from vivarium.examples.disease_model import get_model_specification_path
+
+   p = get_model_specification_path()
+   sim = initialize_simulation_from_model_specification(p)
+
+
+This function returns a simulation object that has not been setup yet so we can
+alter the configuration programmatically, if we wish. Let's alter the
+population size to be smaller so the simulation takes less time to run.
+
+.. code-block:: python
+   sim.configuration.update({'population': {'population_size': 1_000}})
+
+We then need to call the :meth:`setup` method on the simulation context to
+prepare it to run.
+
+.. code-block:: python
+   sim.setup()
+
+After this step, we are ready to  :ref:`run the simulation <interactive_run>`.
+
+.. note::
+   While this is a kind of trivial example, this last use case is extremely
+   important. Practically speaking, the utility of initializing the simulation
+   without setting it up is that it allows you to alter the configuration data
+   and components in the simulation before it is run or examined. This is
+   frequently useful for setting up specific configurations for validating the
+   simulation from a notebook or for reproducing a particular set of
+   configuration parameters that produce unexpected outputs.
+
+.. testcode::
+   from vivarium import initialize_simulation_from_model_specification
+   from vivarium.examples.disease_model import get_model_specification_path
+
+   p = get_model_specification_path()
+   sim = initialize_simulation_from_model_specification(p)
+   sim.configuration.update({'population': {'population_size': 1_000}})
+   sim.setup()
+
+Bonus: Adding Additional Components
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Another use case for using
+:func:`initialize_simulation_from_model_specification <vivarium.interface.interactive.initialize_simulation_from_model_specification>`
+is to extend existing models.
+
+For example, say we wanted to add another risk for unsafe water sources
+into our disease model. We could do the following.
+
+.. testcode::
+
+   from vivarium import initialize_simulation_from_model_specification
+   from vivarium.examples.disease_model import get_model_specification_path, Risk, DirectEffect
+
+   p = get_model_specification_path()
+   sim = initialize_simulation_from_model_specification(p)
+
+   sim.add_components([Risk('unsafe_water_source'),
+                       DirectEffect('unsafe_water_source', 'infected_with_diarrhea.incidence_rate')])
+
+   sim.configuration.update({
+       'unsafe_water_source': {
+           'proportion_exposed': 0.3
+       },
+       'effect_of_unsafe_water_source_on_infected_with_diarrhea.incidence_rate': {
+           'relative_risk': 8,
+       },
+   })
+
+   sim.setup()
+
+This is an easy way to take an old model and toy around with new components
+to immediately see their effects.
+
+
+Exploring the Simulation, Part 1
```

Comment:
I think we discussed previously making this a different tutorial altogether. I can see it here, though. This is the chief way simulations will be explored. The only argument I see against it is wanting to avoid too long of a tutorial